### PR TITLE
Raw proc

### PR DESCRIPTION
**Pull request by jmfranck**
add example to proc scripts that uses the `postproc_type` and `coherence_pathway` properties to show the raw data.

### CODE REVIEW

File: examples/proc_raw.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    image_or_plot(d)
+    if "nScans" in d.dimlabels:
+        d.mean("nScans")
+        fl.next("signal averaged along nScans")
+        image_or_plot(d)
+    if d.get_prop("coherence_pathway") is not None:
+        forplot = abs(d)
+        for thisdim in d.get_prop("coherence_pathway").keys():
+            forplot.sum(thisdim)
```
Comment (guinneak198) [inline examples/proc_raw.py:69]: why/I don't understand why we would sum the coherence pathway? (there's no documentation description for nddata.sum in the documentation)

File: pyspecProcScripts/load_data.py [hunk 2]
```diff
@@ -287,10 +287,24 @@ def proc_bruker_CPMG_v1(s, fl=None):
...
+    return s
+def proc_spincore_diffph_SE_v1(s, fl=None):
+    r"""this one uses a phase cycle where the overall phase and 90-180
+    phase difference are cycled in a nested way -- see the DCCT paper to
+    understand this!"""
+    s.ft(["ph2","ph_diff"], unitary=True)
+    # after the FT, these have a different meaning in terms of coherence
+    # pathways -- remember that when labeling, pySpecData will change the
+    # ph here to a δp
+    s.rename("ph2","ph_overall") # overall change in coherence
```
Comment (guinneak198) [inline pyspecProcScripts/load_data.py:302]: why not just call this ph_overall in the actual pulse sequence before saving?

**Review by guinneak198 — COMMENTED** [review]
AG leaves questions

File: examples/proc_raw.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+Show data with postproc
+====================
+`py proc_raw_data.py NODENAME FILENAME EXP_TYPE`
+
+Fourier transforms (and any needed data corrections for older data) are performed according to the `postproc_type` attribute of the data node.
+This script plots the result, as well as signal that's averaged along the `nScans` dimension.
+
+Tested with:
+
+``py proc_raw.py echo_6 240620_200uM_TEMPOL_pm_echo.h5 ODNP_NMR_comp/Echoes``
```
Comment (guinneak198) [inline examples/proc_raw.py:11]: This doesn't work for me I get "I caught and error but am plotting anyways and nothing shows up in the resulting window

Comment (jmfranck) [reply examples/proc_raw.py:11]: Then debug

Comment (jmfranck) [reply examples/proc_raw.py:11]: note that this does work OK on my laptop (remotely) and on the NMR computer.

File: examples/proc_raw.py (hunk 1 above)
Comment (jmfranck) [reply examples/proc_raw.py]: this was a good comment -- this is fixed in the updated code.

File: examples/proc_raw.py (hunk 3 above)
Comment (guinneak198) [reply examples/proc_raw.py:11]: needed to pull pyspecdata updates

File: examples/proc_raw.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        fl.next("signal averaged along nScans")
+        image_or_plot(d)
+    if d.get_prop("coherence_pathway") is not None:
+        fl.next("sum of abs of all coherence pathways (for comparison)")
+        forplot = abs(d)
+        guess_direct = d.shape.max() # guess that the longest dimension is the direct
+        if guess_direct == 'nScans':
+            temp = d.shape
+            temp.pop('nScans')
+            guess_direct = temp.max()
```
Comment (guinneak198) [inline examples/proc_raw.py:77]: how would it be possible for guess_direct to be nScans if in the block before we average that dimension?

**Review by guinneak198 — COMMENTED** [review]
left a question for discussion

File: examples/proc_raw.py (hunk 4 above)
Comment (jmfranck) [reply examples/proc_raw.py:77]: Reviews are not appropriate forums for discussion

Comment (jmfranck) [reply examples/proc_raw.py:77]: But I will say that the max method of the ndshape object is new, and you should go check it out, consider adding a PR with doc strings

**Comment by jmfranck** [comment]
we reviewed and merged

**Pull request merged**